scripts/seed-supabase-test-user.js

```
@@ -0,0 +1,149 @@
+      { email: TEST_EMAIL, password: TEST_PASSWORD },
+      { headers: { apikey: SUPABASE_KEY, 'Content-Type': 'application/json' } }
+    );
+    userId = data.user?.id;
+    if (userId) {
+      console.log('User registered:', userId);
+    } else {
+      throw new Error('No user id returned');
+    }
+  } catch (err) {
+    // If user exists, fetch user id via admin API
+    if (err.response && err.response.data?.msg?.includes('User already registered')) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)